galleries/examples/color/individual_colors_from_cmap.py

```diff
@@ -0,0 +1,61 @@
+"""
+===========================================
+Selecting individual colors from a colormap
+===========================================
+
+Sometimes we want to use more colors or a different set of colors than the default color
+cycle provides. Selecting individual colors from one of the provided colormaps can be a
+convenient way to do this.
+
+Once we have hold of a `.Colormap` instance, the individual colors can be accessed
+simply using its ``.colors`` attribute.  If we want a specific number of colors taken
+at regular intervals from a continuous colormap, we can create a new colormap using the
+`~.Colormap.resampled` method.
+"""
+
+import matplotlib.pyplot as plt
+
+import matplotlib as mpl
+
+n_lines = 21
+
+cmap = mpl.colormaps.get_cmap('plasma').resampled(n_lines)
+colors = cmap.colors
+
+fig, ax = plt.subplots(layout='constrained')
+
+for i, color in enumerate(colors):
+    ax.plot([0, i], color=color)
+
+plt.show()
+
+# %%
+# Alternatively, we may want to replace the default color cycle with a different
+# discrete set of colors.  We can use a `~cycler.cycler` instance within `.rcParams` to
+# achieve that.  See :ref:`color_cycle` for details.
+
+
+from cycler import cycler
+
+cmap = mpl.colormaps.get_cmap('Dark2')
+
+with mpl.rc_context({'axes.prop_cycle': cycler(color=cmap.colors)}):
+
+    fig, ax = plt.subplots(layout='constrained')
+
+    for i in range(n_lines):
+        ax.plot([0, i])
+
+plt.show()
+
+# %%
+#
+# .. admonition:: References
+#
+#    The use of the following functions, methods, classes and modules is shown
+#    in this example:
+#
+#    - `matplotlib.colormaps.get_cmap`
+#    - `matplotlib.colors.Colormap`
+#    - `matplotlib.colors.Colormap.resampled`
+#    - `maplotlib.rc_context`
```

Comment:
In my local build, `get_cmap` and `rc_context` failed to make a link and I have no idea why.